tests/utils/mock-api.js

```diff
@@ -19,7 +20,7 @@ const startMockApi = ({ routes }) => {
   routes.forEach(({ method = 'get', path, response = {}, status = 200, requestBody }) => {
     app[method.toLowerCase()](`/api/v1/${path}`, function onRequest(req, res) {
       // validate request body
-      if (requestBody !== undefined && requestBody !== JSON.stringify(req.body)) {
+      if (requestBody !== undefined && !equal(requestBody, req.body)) {
```

Comment:
Would [`util.isDeepStrictEqual()`](https://nodejs.org/api/util.html#util_util_isdeepstrictequal_val1_val2) work? This is supported now that we've dropped Node 8 from the CLI.

Reply:
Yes! I thought about using `assert.deepStrictEqual` but didn't want a method that throws. Didn't know about `util.isDeepStrictEqual` I'll give it a try

Reply:
Done in https://github.com/netlify/cli/pull/2378/commits/3937df4a05dc65878db32a515c6c140f1fd03d2d